setup_project_with_framework.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/sh
+
+startdir=$(pwd)
+
+cmssw_arch="slc6_amd64_gcc530" ## should be in sync with CMSSW.arch
+cmssw_version="CMSSW_8_0_30"   ## should be in sync with CMSSW.release
```

Comment:
why not read it from `CMSSW.arch` and `CMSSW.release` files then ? This would again avoid having to do it twice ?
I see your warnings below but I am not sure to see the use case, is it for whenever we would want to test an unstable recipe before it is merged ?

Reply:
That's easy to do, but I wanted to make the "initial setup" as light as possible, and now this is done before the Framework is cloned. But there is a solution to that: I can clone it to a temporary directory, then read them from there, do everything up to the bootstrap, and then move it into place. I will update this

Reply:
Having this for unstable recipes would be nice, I'll see if it is easy to implement by taking a single argument